specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2020-01-01/iotdps.json

```diff
@@ -1418,6 +1418,11 @@
           "type": "boolean",
           "readOnly": true
         },
+        "certificate": {
+          "description": "base-64 representation of X509 certificate .cer file or just .pem file content.",
```

Comment:
If this is a base64-encoded string, then this should also have ```"format": "byte"```

Reply:
I have included the fix in the latest commit.